### PR TITLE
Add external dependencies used only in MVC benchmarks apps

### PR DESCRIPTION
**Pull request by dougbu**
- part of aspnet/Mvc#7805

### CODE REVIEW

**Comment by dougbu** [comment]
See also aspnet/Mvc#7944

**Comment by dougbu** [comment]
928a81155e

File: build/dependencies.props [hunk 1]
```diff
@@ -49,6 +49,19 @@
...
+
+    <!--
+      BenchmarksOnly* package versions come from NuGet.org and are intended only for use in benchmarks apps where EF
+      is not otherwise referenced. They avoid unnecessary changes to the Universe build graph or to product
+      dependencies. Do not use these properties elsewhere.
+    -->
+    <BenchmarksOnlyMicrosoftEntityFrameworkCoreDesignPackageVersion>2.1.0</BenchmarksOnlyMicrosoftEntityFrameworkCoreDesignPackageVersion>
+    <BenchmarksOnlyMicrosoftEntityFrameworkCoreSqlitePackageVersion>2.1.0</BenchmarksOnlyMicrosoftEntityFrameworkCoreSqlitePackageVersion>
+    <BenchmarksOnlyMicrosoftEntityFrameworkCoreSqlServerPackageVersion>2.1.0</BenchmarksOnlyMicrosoftEntityFrameworkCoreSqlServerPackageVersion>
+    <BenchmarksOnlyMySqlConnectorPackageVersion>0.42.1</BenchmarksOnlyMySqlConnectorPackageVersion>
```
Comment (natemcmaster) [inline build/dependencies.props]: Just double checking. Did we get OSS approval to use these third-party packages? (cc @Eilon - just to confirm, this required for test-only dependencies, right?)

**Comment by dougbu** [comment]
>  just to confirm, this required for test-only dependencies, right?

Yup, 100 percent test-only. These packages are already used in the aspnet/benchmarks repo. What's new is using them anywhere in a Universe-controlled build (aspnet/benchmarks builds on the same CI but not under Universe).

File: build/dependencies.props (hunk 1 above)
Comment (Eilon) [reply build/dependencies.props]: OSS approval is required for all OSS usage anywhere, for any reason.

Comment (dougbu) [reply build/dependencies.props]: @Eilon absolutely. Are you saying we didn't get approval before using these packages in aspnet/benchmarks?

Comment (Eilon) [reply build/dependencies.props]: I only said what I said. I didn't say anything else 😄

**Pull request closed without merging**